src/views/Zap/__tests__/ZapStakeAction.unit.test.tsx

```diff
@@ -204,95 +119,24 @@ describe("<ZapStakeAction/> ", () => {
     fireEvent.click(await screen.getByText("Approve"));
     expect(await screen.findByText("Successfully approved")).toBeInTheDocument();
   });
-
-  it.skip("Should Display Error when unable to retrieve allowances", async () => {
-    // Contract throws an error when trying to get the allowance
-    Contract.IERC20__factory.connect = jest.fn().mockReturnValue({
-      allowance: jest.fn().mockImplementation(() => {
-        throw new Error("Error");
-      }),
-      symbol: jest.fn().mockReturnValue("DAI"),
-    });
-
-    render(
-      <>
-        <Messages />
-        <ZapStakeAction />
-      </>,
-    );
-    fireEvent.click(await screen.findByTestId("zap-input"));
-    fireEvent.click(await screen.getAllByText("DAI")[0]);
-    expect(await screen.findByText("An error has occurred when fetching token allowance."));
-  });
-
-  it.skip("Should Display Error when unable to approve allowance", async () => {
```

Comment:
Also worth testing

Reply:
Same note as above. If you've got these written working w/ TokenAllowanceGuard, they can for sure be included.